app/models/position.rb

```diff
@@ -12,6 +12,16 @@ class Position < ApplicationRecord
 
   validates_presence_of :course_code, :openings, :session
   validates :openings, numericality: { only_integer: true }
+
+  def as_json(_options = {})
+    super(
+      except: [:session_id],
```

Comment:
Shouldn't we use `%i[session_id]`?

I thought the linter would catch this.

Reply:
guess linter doesn't complain if it's just one element, but will update